The biosynthetic pathway of target phytochemicals was analyzed using the KEGG (Kyoto Encyclopedia of Genes and Genomes) database. The KEGG PATHWAY used to 
identify enzymes and genes involved in the biosynthesis of specific secondary metabolites. Genomic and protein sequences of these genes were downloaded in FASTA format for downstream analysis. 
Sequence alignment, open reading frame prediction, and codon optimization were performed using Biopython modules such as Bio.Seq and Bio.SeqIO. For recombinant DNA design, gene cloning sites were determined computationally, and 
appropriate vectors were selected. Restriction enzyme sites were analyzed using the Biopython Bio. Restriction package to facilitate In Silico cloning. All computational experiments were 
conducted in a Python environment using Jupyter Notebook and PyCharm. Plasmid design and visualization were carried out using Snap Gene Viewer, where the synthesized gene insert was cloned in-silico into a 
suitable expression vector (e.g., pET, pUC, or pCAMBIA vectors, depending on the host system). 
